pkg/cmd/server/api/validation/master.go

```diff
@@ -287,6 +287,18 @@ func ValidateAssetConfig(config *api.AssetConfig) fielderrors.ValidationErrorLis
 		allErrs = append(allErrs, urlErrs...)
 	}
 
+	if len(config.LoggingPublicURL) > 0 {
+		if _, loggingURLErrs := ValidateURL(config.LoggingPublicURL, "loggingPublicURL"); len(loggingURLErrs) > 0 {
```

Comment:
@spadgett Yep, the latest deployer currently creates a normal route, but that will need to be updated to a secured one. Is pass through fine or do you need re-encrypt?